### PR TITLE
fix example command downgrade ssa to csa

### PR DESCRIPTION
**Pull request by roelarents**
The command to downgrade from server side apply to client side apply is exactly the same as supplying a non-default field manager. I suppose it was copy/pasted and the `field-manager` option was not altered. If I read correctly, the field manager should be set (back) to `kubectl` to downgrade.

(replaces #28681 because that was signed with a noreply email, sorry)

### CODE REVIEW

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 1]
```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
 manager for kubectl server-side apply is `kubectl`.
```
Comment (sftim) [inline content/en/docs/reference/using-api/server-side-apply.md:459]: We might want to explain when you would want to set a different value than `kubectl`. Neither the existing nor the new doc make that clear.

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 2]
```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
...
 
 ```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
+kubectl apply --server-side --field-manager=kubectl [--dry-run=server]
```
Comment (apelisse) [inline content/en/docs/reference/using-api/server-side-apply.md:462]: I don't know if I agree, the sentence before that reads:
> As an exception, you can opt-out of this behavior by specifying a different, non-default field manager, as seen in the following example.

It says the example will use a non-default field manager (so NOT `kubectl`). Or am I misreading?

But I think stfim is making a better point, the thing is unclear in general.

**Comment by apelisse** [comment]
@roelarents Thanks for looking at our docs and using the feature! ;-)

What were you trying to do, trying to understand, and how did it fail or confuse you?

File: content/en/docs/reference/using-api/server-side-apply.md (hunk 2 above)
Comment (kbhawkey) [reply content/en/docs/reference/using-api/server-side-apply.md:462]: Hi @roelarents .
I agree that this section is confusing.

Can you use the sample kubectl command (`kubectl apply --server-side --field-manager=my-manager [--dry-run=server]`) to downgrade/switch from server-side apply to client-side apply?

**Comment by roelarents** [comment]
i was trying to read up about server-side apply and it confused me that _Upgrading from client-side apply to server-side apply_ and _Downgrading from server-side apply to client-side apply_ have the same example command. However after reading again, especially the paragraph above the command, that command seems not an example on how to upgrade or downgrade, but on how to use a custom field-manager. Then still, I would change the example command in the _downgrading to client-side_ chapter, because it includes the `--server-side` flag, which if I understand correctly is what you should omit if you want to downgrade "back" to client-side.

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 3]
```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
 manager for kubectl server-side apply is `kubectl`.
 
 ```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
+kubectl apply --field-manager=my-manager [--dry-run=server]
```
Comment (kbhawkey) [inline content/en/docs/reference/using-api/server-side-apply.md:462]: @apelisse , Do you have time to review this change? Thank you.

Comment (apelisse) [reply content/en/docs/reference/using-api/server-side-apply.md:462]: Yeah, I still don't know if I'm convinced at least that this command is better.

The text above reads:
> This behavior applies to server-side apply with the `kubectl` field manager.
As an exception, you can opt-out of this behavior by specifying a different,
non-default field manager, as seen in the following example. The default field
manager for kubectl server-side apply is `kubectl`.

"if you want to server-side apply the non-default manager, you can do ..."
and then the command is right. The whole thing might be confusing or not helpful, 
but AFAIU the command is exactly what the text says.

**Comment by jimangel** [comment]
@roelarents are you still willing to work on this? I'm going to close the PR given the duration it's been open with no work since Aug 2021, but feel free to reopen if you pick it back up! Thanks!
/close

**Pull request closed without merging**